chrome-launcher/chrome-launcher.ts

```diff
@@ -237,11 +241,19 @@ export class Launcher {
             })
             .catch(err => {
               if (retries > 10) {
+                log.error('ChromeLauncher', err.message);
```

Comment:
Yes. But since it's the most important part of that wall-of-text it seems ok. :)